schemas/2015-01-01/deploymentTemplate.json

```diff
@@ -536,7 +536,8 @@
                   { "$ref": "https://schema.management.azure.com/schemas/2018-03-01/Microsoft.Insights.json#/resourceDefinitions/actionGroups" },
                   { "$ref": "https://schema.management.azure.com/schemas/2018-03-01/Microsoft.BatchAI.json#/resourceDefinitions/clusters" },
                   { "$ref": "https://schema.management.azure.com/schemas/2018-03-01/Microsoft.BatchAI.json#/resourceDefinitions/fileServers" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2018-03-01/Microsoft.BatchAI.json#/resourceDefinitions/jobs" }
+                  { "$ref": "https://schema.management.azure.com/schemas/2018-03-01/Microsoft.BatchAI.json#/resourceDefinitions/jobs" },
+                  { "$ref": "https://schema.management.azure.com/schemas/2018-03-01/Microsoft.Insights.json#resourceDefinitions/metricAlerts" }
```

Comment:
>{ "$ref": "https://schema.management.azure.com/schemas/2018-03-01/Microsoft.Insights.json#resourceDefinitions/metricAlerts" } [](start = 18, length = 125)

can you put this to the line right below actionGroups?

Reply:
Done.